### PR TITLE
validate: isbn

### PR DESCRIPTION
**Pull request by bangarang**
# @flatfile/plugin-validate-isbn

Validates ISBN numbers!

### CODE REVIEW

File: validate/isbn/package.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  },
+  "keywords": [
+    "flatfile",
+    "plugin",
+    "isbn",
+    "validator",
+    "flatfile-plugins",
+    "category-transform"
+  ],
+  "author": "Your Name",
```
Comment (carlbrugger) [inline validate/isbn/package.json:43]:
```suggestion
  "author": "Flatfile, Inc..",
```

File: validate/isbn/package.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  "license": "MIT",
```
Comment (carlbrugger) [inline validate/isbn/package.json:44]:
```suggestion
  "license": "ISC",
```

File: validate/isbn/package.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    "@flatfile/listener": "^1.1.0"
+  },
+  "devDependencies": {
+    "@flatfile/rollup-config": "^0.1.1",
+    "@flatfile/utils-testing": "^0.3.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FlatFilers/flatfile-plugins.git",
+    "directory": "plugins/isbn-validator"
```
Comment (carlbrugger) [inline validate/isbn/package.json:59]:
```suggestion
    "directory": "validate/isbn-validator"
```

File: validate/isbn/package.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@flatfile/plugin-validate-isbn",
+  "version": "1.0.0",
+  "description": "A Flatfile Listener plugin for ISBN validation with configurable options. Validates ISBN-10 and ISBN-13 formats, provides specific error messages, and optionally converts ISBN-10 to ISBN-13.",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "browser": {
+    "./dist/index.js": "./dist/index.browser.js",
```
Comment (carlbrugger) [inline validate/isbn/package.json:9]:
```suggestion
    "./dist/index.cjs": "./dist/index.browser.cjs",
```

File: validate/isbn/package.json [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    "./dist/index.mjs": "./dist/index.browser.mjs"
+  },
+  "exports": {
+    "types": "./dist/index.d.ts",
+    "node": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
```
Comment (carlbrugger) [inline validate/isbn/package.json:16]:
```suggestion
      "require": "./dist/index.cjs"
```

File: validate/isbn/package.json [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    },
+    "browser": {
+      "require": "./dist/index.browser.js",
```
Comment (carlbrugger) [inline validate/isbn/package.json:19]:
```suggestion
      "require": "./dist/index.browser.cjs",
```

File: validate/isbn/package.json [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    "test": "jest ./**/*.spec.ts --config=../../jest.config.js --runInBand"
+  },
+  "keywords": [
+    "flatfile",
+    "plugin",
+    "isbn",
+    "validator",
+    "flatfile-plugins",
+    "category-transform"
+  ],
```
Comment (carlbrugger) [inline validate/isbn/package.json:42]:
```suggestion
  "keywords": [
    "flatfile-plugins",
    "category-validate"
  ],
```

File: validate/isbn/package.json [hunk 8]
```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@flatfile/plugin-validate-isbn",
+  "version": "1.0.0",
+  "description": "A Flatfile Listener plugin for ISBN validation with configurable options. Validates ISBN-10 and ISBN-13 formats, provides specific error messages, and optionally converts ISBN-10 to ISBN-13.",
+  "main": "./dist/index.js",
```
Comment (carlbrugger) [inline validate/isbn/package.json:5]:
```suggestion
  "main": "./dist/index.cjs",
```

File: validate/isbn/package.json [hunk 9]
```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@flatfile/plugin-validate-isbn",
+  "version": "1.0.0",
```
Comment (carlbrugger) [inline validate/isbn/package.json:3]:
```suggestion
  "version": "0.0.0",
```

File: validate/isbn/rollup.config.mjs [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+  includeUmd: true,
+  umdConfig: {
+    name: 'FlatfileIsbnValidator',
+    globals: {
+      '@flatfile/listener': 'FlatfileListener',
+      '@flatfile/plugin-record-hook': 'FlatfilePluginRecordHook',
+      'isbn3': 'ISBN'
+    }
+  }
+});
```
Comment (carlbrugger) [inline validate/isbn/rollup.config.mjs:15]:
```suggestion
const config = buildConfig({});

export default config;
```

**Review by carlbrugger — APPROVED** [review]
Approved with a few nits

File: validate/isbn/package.json [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+    "@flatfile/listener": "^1.1.0"
+  },
+  "devDependencies": {
+    "@flatfile/rollup-config": "^0.1.1",
+    "@flatfile/utils-testing": "^0.3.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FlatFilers/flatfile-plugins.git",
+    "directory": "validate/isbn-validator"
```
Comment (bangarang) [inline validate/isbn/package.json:55]:
```suggestion
    "directory": "validate/isbn"
```

File: validate/isbn/README.MD [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+import { FlatfileListener } from "@flatfile/listener";
+import isbnValidator from "@flatfile/plugin-isbn-validator";
+
+const listener = new FlatfileListener();
+
+listener.use(
+  isbnValidator({
+    sheetSlug: "books",
+    isbnFields: ["isbn", "alternate_isbn"],
+    autoConvert: true,
```
Comment (bangarang) [inline validate/isbn/README.MD:41]:
```suggestion
    format: "isbn13",
```

**Pull request merged**